src/main/fc/fc_core.c

```diff
@@ -975,7 +974,7 @@ void taskMainPidLoop(timeUs_t currentTimeUs)
     }
 #endif
     // Check if landed, FW and MR
-    if (STATE(ALTITUDE_CONTROL)) {
+    if (STATE(ALTITUDE_CONTROL) || isFwLandInProgess()) {
```

Comment:
This change isn't required surely given `STATE(ALTITUDE_CONTROL)` will always be true if an Airplane platform is configured.